group1/requirements/views/projects.py

```diff
@@ -266,9 +267,11 @@ def get_attachments(request, projectID):
 
 @user_can_access_project()
 def upload_attachment(request, projectID):
-
+    print(request.FILES['file'].name);
```

Comment:
Is there a purpose to this print statement besides debugging?  If so, maybe more context can be provided for clarity instead of just outputting a file name here.  Otherwise, maybe this print statement can be removed?